<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>framelessWidgetDeveloper</class>
 <widget class="QWidget" name="framelessWidgetDeveloper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="mainWidget" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget { border: none; }</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="mainLayout" stretch="1,20">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="titleBar" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#titleBar{

background-color: #6bc8cc;}</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <property name="topMargin">
                <number>6</number>
               </property>
               <property name="bottomMargin">
                <number>6</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <family>Arial Black</family>
                   <pointsize>11</pointsize>
                   <bold>true</bold>
                   <underline>false</underline>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel{

font-weight:bold;
color:white;
border:none;
}</string>
                 </property>
                 <property name="text">
                  <string>创作者模式</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>600</width>
                   <height>13</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="Button_min">
                 <property name="minimumSize">
                  <size>
                   <width>18</width>
                   <height>18</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>18</width>
                   <height>18</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{background-color:#c2c2c2;border-radius:9px}
QPushButton:hover {background-color:#f9bf45;border-radius:9px;}
QPushButton:pressed {background-color:#ffb11b;border-radius:9px;}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QPushButton" name="Button_max">
                 <property name="minimumSize">
                  <size>
                   <width>18</width>
                   <height>18</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>18</width>
                   <height>18</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{background-color:#c2c2c2;border-radius:9px}
QPushButton:hover {background-color:#227d51;border-radius:9px;}
QPushButton:pressed {background-color:#2d6d4b;border-radius:9px;}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QPushButton" name="Button_close">
                 <property name="minimumSize">
                  <size>
                   <width>18</width>
                   <height>18</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>18</width>
                   <height>18</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{background-color:#c2c2c2;border-radius:9px}
QPushButton:hover {background-color:#cb4042;border-radius:9px;}
QPushButton:pressed {background-color:#ab3b3a;border-radius:9px;}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="mainDisplayWidget" native="true">
              <property name="styleSheet">
               <string notr="true">#mainDisplayWidget{
background-color: rgb(251, 251, 251);}</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3" columnstretch="3,0">
               <property name="leftMargin">
                <number>3</number>
               </property>
               <property name="topMargin">
                <number>3</number>
               </property>
               <property name="rightMargin">
                <number>3</number>
               </property>
               <property name="bottomMargin">
                <number>3</number>
               </property>
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <property name="leftMargin">
                  <number>5</number>
                 </property>
                 <property name="bottomMargin">
                  <number>5</number>
                 </property>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QtMaterialFlatButton" name="new_story">
                     <property name="styleSheet">
                      <string notr="true">  QPushButton {
      border: 0.5px solid white;
      border-radius: 6px;
      background-color: rgb(90,194,198);
      min-width: 90px;
	  font-family: &quot;Microsoft YaHei&quot;;
	  font-size:11pt;
	  font-weight: bold;
	  color:white;
  }
 QPushButton:hover {
	border: 0.5px solid white;
      border-radius: 6px;
      background-color: #1fab89;
      min-width: 80px;
	  font-family: &quot;Microsoft YaHei&quot;;
	  font-size:10pt;
	  font-weight: bold;
	  color:white;
 }</string>
                     </property>
                     <property name="text">
                      <string>新建剧情节点</string>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QtMaterialFlatButton" name="new_childstory">
                     <property name="styleSheet">
                      <string notr="true">  QPushButton {
      border: 0.5px solid white;
      border-radius: 6px;
      background-color: rgb(90,194,198);
      min-width: 90px;
	  font-family: &quot;Microsoft YaHei&quot;;
	  font-size:11pt;
	  font-weight: bold;
	  color:white;
  }
 QPushButton:hover {
	border: 0.5px solid white;
      border-radius: 6px;
      background-color: #1fab89;
      min-width: 80px;
	  font-family: &quot;Microsoft YaHei&quot;;
	  font-size:10pt;
	  font-weight: bold;
	  color:white;
 }</string>
                     </property>
                     <property name="text">
                      <string>新建子剧情节点</string>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QtMaterialFlatButton" name="new_choice">
                     <property name="styleSheet">
                      <string notr="true">  QPushButton {
      border: 0.5px solid white;
      border-radius: 6px;
      background-color: rgb(90,194,198);
      min-width: 90px;
	  font-family: &quot;Microsoft YaHei&quot;;
	  font-size:11pt;
	  font-weight: bold;
	  color:white;
  }
 QPushButton:hover {
	border: 0.5px solid white;
      border-radius: 6px;
      background-color: #1fab89;
      min-width: 80px;
	  font-family: &quot;Microsoft YaHei&quot;;
	  font-size:10pt;
	  font-weight: bold;
	  color:white;
 }</string>
                     </property>
                     <property name="text">
                      <string>新建选项节点</string>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QtMaterialFlatButton" name="new_delete">
                     <property name="styleSheet">
                      <string notr="true">  QPushButton {
      border: 0.5px solid white;
      border-radius: 6px;
      background-color: rgb(90,194,198);
      min-width: 90px;
	  font-family: &quot;Microsoft YaHei&quot;;
	  font-size:11pt;
	  font-weight: bold;
	  color:white;
  }
 QPushButton:hover {
	border: 0.5px solid white;
      border-radius: 6px;
      background-color: #1fab89;
      min-width: 80px;
	  font-family: &quot;Microsoft YaHei&quot;;
	  font-size:10pt;
	  font-weight: bold;
	  color:white;
 }</string>
                     </property>
                     <property name="text">
                      <string>删除节点</string>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QGraphicsView" name="graphicsView">
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="0" column="1">
                <widget class="QTabWidget" name="edit_value">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="sizeIncrement">
                  <size>
                   <width>200</width>
                   <height>360</height>
                  </size>
                 </property>
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>16</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="elideMode">
                  <enum>Qt::ElideNone</enum>
                 </property>
                 <property name="tabsClosable">
                  <bool>false</bool>
                 </property>
                 <property name="movable">
                  <bool>false</bool>
                 </property>
                 <property name="tabBarAutoHide">
                  <bool>false</bool>
                 </property>
                 <widget class="QWidget" name="edit_valuePage2">
                  <attribute name="title">
                   <string/>
                  </attribute>
                  <layout class="QGridLayout" name="gridLayout_5">
                   <item row="1" column="0">
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                     <item>
                      <widget class="QLabel" name="node_attribute_label">
                       <property name="text">
                        <string>属性编辑</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout">
                       <item>
                        <widget class="QLineEdit" name="attribute_name_Edit">
                         <property name="text">
                          <string>属性名称</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QSpinBox" name="set_attribute_value"/>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
                       <property name="spacing">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QtMaterialFlatButton" name="add_attribute_button">
                         <property name="text">
                          <string>增加</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QtMaterialFlatButton" name="delete_attribute_button">
                         <property name="text">
                          <string>删除</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <widget class="QGraphicsView" name="graphicsView_2"/>
                     </item>
                    </layout>
                   </item>
                   <item row="0" column="0">
                    <layout class="QVBoxLayout" name="verticalLayout_5">
                     <item>
                      <widget class="QLabel" name="node_label">
                       <property name="text">
                        <string>节点编辑</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QTextEdit" name="node_text_Edit">
                       <property name="html">
                        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;文本内容编辑&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QtMaterialFlatButton" name="save_text_button">
                       <property name="text">
                        <string>保存文本内容</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="edit_valuePage1">
                  <attribute name="title">
                   <string/>
                  </attribute>
                  <widget class="QLabel" name="label_4">
                   <property name="geometry">
                    <rect>
                     <x>60</x>
                     <y>20</y>
                     <width>81</width>
                     <height>16</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>属性编辑</string>
                   </property>
                  </widget>
                  <widget class="QPushButton" name="pushButton_4">
                   <property name="geometry">
                    <rect>
                     <x>20</x>
                     <y>50</y>
                     <width>80</width>
                     <height>24</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>增加属性</string>
                   </property>
                  </widget>
                  <widget class="QPushButton" name="pushButton_5">
                   <property name="geometry">
                    <rect>
                     <x>140</x>
                     <y>50</y>
                     <width>80</width>
                     <height>24</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>删除属性</string>
                   </property>
                  </widget>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialFlatButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialflatbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
